t/plugin/udp-logger.t

```diff
@@ -347,6 +348,12 @@ GET /t
                 return
             end
             ngx.say(body)
+            local code, _, _ = t("/hello", "GET")
+            if code >= 300 then
+                ngx.status = code
+                ngx.say("fail")
+                return
+            end
```

Comment:
The test case's name is `add plugin`, but you also access the route in the test case, why not create a new test case to access? 
```
=== TEST 10: access
```

Reply:
renamed the tests appropriately.